cilium/cmd/cleanup.go

```diff
@@ -133,6 +136,22 @@ func removeCNI() error {
 	return err
 }
 
+func unmountCgroup() error {
+	cgroupRoot := defaults.DefaultCgroupRoot
+	cgroupRootStat, err := os.Stat(cgroupRoot)
+	if err != nil {
```

Comment:
If the cgroupRoot was never mounted, maybe because cgroupv2 is not supported, it looks like this will print an error. How about if the cgroupRoot does not exist returning nil with the assumption that it was never created in the first place?

Reply:
True thats nicer @jrfastab, resolved